types/convert_test.go

```diff
@@ -281,6 +281,14 @@ func (s *testTypeConvertSuite) TestConvertType(c *C) {
 	c.Assert(err, IsNil)
 	_, err = Convert(bj2, ft)
 	c.Assert(err, NotNil)
+	bj3, err := json.ParseBinaryFromString("{\"key\": 99}")
+	c.Assert(err, IsNil)
+	_, err = Convert(bj3, ft)
+	c.Assert(err, NotNil)
+	bj4, err := json.ParseBinaryFromString("[99 00 01]")
+	c.Assert(err, NotNil)
```

Comment:
if parse errors, it's meaningless to covert bj4?
```
bj4, err := json.ParseBinaryFromString("[99,0,1]")
c.Assert(err, IsNil)
```

Reply:
Fixing...